webserver.py

```diff
@@ -0,0 +1,60 @@
+import tornado.ioloop
+import tornado.web
+import MySQLdb
+import calendar, time
+import json
+
+class MainHandler(tornado.web.RequestHandler):
+
+	def get(self):
+		self.render("index.html")
+
+class ResourceHandler(tornado.web.RequestHandler):
+
+	def get(self, source):
+		resource = open(source).read()
+		self.set_header("Content-Type", 'text/css; charset="utf-8"')
+		self.write(resource)
+
+class DataHandler(tornado.web.RequestHandler):
+
+	def get(self, coin, change):
+		self.db = MySQLdb.connect('localhost', 'coingraphs', 'CGpassword', 'coingraphs')
+		c  = self.db.cursor()
+
+		if coin != "eth" and coin != "btc": # Quick dirty workaround for sanitization
+			self.write('{"message":"invalid_coin"}')
+		c.execute("SELECT time, %s AS price FROM "+coin+" WHERE time > %s ORDER BY time ASC",
+					   (change, calendar.timegm(time.gmtime()) - 36000))
+
+		row_headers=[x[0] for x in c.description]
+		data = c.fetchall()
+		print(data)
+		json_data = []
+		for row in data:
+			json_data.append(dict(zip(row_headers,row)))
+
+		self.write(json.dumps(json_data))
+
+
+def make_app():
+
+	settings = {
+		"debug": True
```

Comment:
It's probably not a good thing to ship it with the debug mode enabled by default, it has security implications (in this case, mostly information disclosure).

Reply:
See #4.